src/services/item/service.ts

```diff
@@ -386,15 +386,20 @@ export class ItemService {
     return filterOutPackedItems(actor, repositories, children);
   }
 
-  async _getDescendants(actor: Actor, repositories: Repositories, itemId: UUID) {
+  async _getDescendants(
```

Comment:
```suggestion
  private async getDescendants(
```

Reply:
I've set it to private, but since there is already a public function getDescendants, I can't rename it `getDescendants`. I think we can rename it into a more accurate name. Any suggestions ?

What about renaming the publics ones to getFilteredDescendants ?

Reply:
Yes, we can rename them, or maybe rename the private ones ?

Reply:
Yes I think it is a good idea to rename it `getDescendants` for the private one and `getFilteredDescendants` for the public one, because it is explain what the functions really do.